site/content/docs/Installation/_index.md

```diff
@@ -266,10 +266,10 @@ to a repository. The names will be persisted in a configuration file and used af
 
 ### Compilation From Source
 
-If you have [Go 1.16](https://golang.org/) or newer, you may download and build Kopia yourself. No special setup is necessary, other than the Go compiler. You can simply run:
+If you have [Go 1.16](https://golang.org/) or newer, you may download and build Kopia yourself. No special setup is necessary, other than the Go compiler and [git](https://git-scm.com/). You can simply run:
 
 ```shell
-$ go get github.com/kopia/kopia
+$ go install github.com/kopia/kopia
```

Comment:
Pretty sure this should be:
```console
$ go install github.com/kopia/kopia@latest
```

Reply:
Hmm..

```
If the arguments don't have version suffixes, "go install" may run in module-aware mode or GOPATH mode, depending on the GO111MODULE environment variable and the presence of a go.mod file. See 'go help modules' for details. If module-aware mode is enabled, "go install" runs in the context of the main module.
```

Reply:
@Shrekster: Correct!

It used to be that `go get` would download and install a package. It was then upgraded to also update `go.mod`.

Go 1.18 is going to continue that set of changes such that `go get` will no longer install anything and will only be used to update `go.mod` (same as using the `-d` flag for `go get`).

Conversely, `go install` has been upgraded to fill the gap left by `go get` no longer installing downloaded packages. The version suffix was added as a way to ignore GOPATH and any local `go.mod` and instead download the specified module and install a package from it into `$GOBIN`.

The `go install` command as written (no version suffix) will install the executable into `$GOBIN`, but it will source the version from a local `go.mod` file. If no `go.mod` file exists or if Kopia isn't present in that `go.mod` file, the installation will fail.

Reply:
```suggestion
$ go install github.com/kopia/kopia@latest
```